entrypoint.sh.release

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 grepzt() {
-  [ ! -n "$(cat /var/lib/zerotier-one/zerotier-one.pid)" -a -d "/proc/$(cat /var/lib/zerotier-one/zerotier-one.pid)" ]
+  [ -n "$(cat /var/lib/zerotier-one/zerotier-one.pid)" ] && [ -d "/proc/$(cat /var/lib/zerotier-one/zerotier-one.pid)" ]
```

Comment:
you have inverted this logic, I think? is it really detecting if the service is up now?